src/style.css

```
@@ -0,0 +1,32 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  list-style: none;
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars__star:nth-child(n) {
+  background-image: url(/images/star.svg);
+}
```

just add background-image to .stars__star, cause each star has this class
.stars__star:nth-child(n) selector basically says 'select each star with .stars__star class'

```suggestion
.stars__star {
  list-style: none;
  height: 16px;
  width: 16px;
  background-repeat: no-repeat;
  background-position: center;
  background-image: url(/images/star.svg);
}

.stars__star:not(:last-child) {
  margin-right: 4px;
}
```

---

now I got it, done